addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.js

```diff
@@ -135,6 +136,27 @@ export class ProductLabelSectionAndNoteField extends Many2OneField {
                 this.labelNode.el.focus();
             }
         });
+
+        this.onBeforePrint = () => {
+            this.setPrintMode(true);
+        };
```

Comment:
```suggestion

        };
```
Btw, we have [some guidelines & tools](https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines) for Javascript in Odoo, if you're curious I'd suggest your to install them eventually.

Reply:
This one seemed weird to me so I spent some time reinstalling the JS Toolings to see what it was about but the linter wasn't spotting anything on that line so I thought maybe you meant the line break to be after the semicolon instead of before the curly bracket ? Please tell me if it's still wrong in your opinion.